.github/workflows/main.yml

```diff
@@ -82,6 +82,12 @@ jobs:
         run: |
           make lint-js
 
+      - name: make test-js
+        env:
+          LARAVEL_BYPASS_ENV_CHECK: '1'
```

Comment:
`laravel-vite-plugin` tries to be helpful by blowing up if it detects the vite server is running in CI. The `LARAVEL_BYPASS_ENV_CHECK` env var disables this check.